src/app/modules/backbone/components/floating-action/floating-action.component.scss

```diff
@@ -0,0 +1,12 @@
+.floating-action-background {
+    @apply fixed flex items-center justify-center z-50;
+    @apply w-full h-full;
+    @apply bg-gray-600 bg-opacity-70;
+}
+
+.floating-action-button {
+    @apply p-4 rounded-full shadow cursor-pointer;
```

Comment:
should add fixed class with right-10, bottom-10 to position in lower right corner of the screen

please rearrange classes with new lines for layout, sizing, presentations, pseudo (hover)

Reply:
Yes, this clarifies it for me—I thought it was supposed to be like an alert in the centre of the screen that dims everything else, but in fact it should be a little pop-up in the corner!

Reply:
Ahhh makes sense why you had it that way initially.

Yes, exactly. The floating button should be visible at (almost) all times.